### ISSUE TITLE
Removal of ads

### ISSUE THREAD

**Issue by JayTeeHaitch** [boukestam/inbox-in-gmail#90]
Is it possible to remove the adds that gmail puts in to look like emails? I was using the "Simplify Gmail" until I discovered this app, it removes the ads so I assume there must be a way to do it. https://chrome.google.com/webstore/detail/simplify-gmail/pbmlfaiicoikhdbjagjbglnbfcbcojpj

**Comment by richardfrost** [comment]
Yes, it is possible. It was actually implemented before, but there was a conscious decision to revert it in [0.5.2](https://github.com/boukestam/inbox-in-gmail/releases/tag/0.5.2) to try and get back on the Chrome Web Store.

**Comment by russelldc** [comment]
Yeah, this was a decision I made, and it's up for debate, really.

We're not sure why it got taken down from the extension store in the first place, so I thought I'd try to eliminate any possibilities. Within the next few weeks, I'll be publishing the extension under my own account after making a few updates. If it gets taken down again, I'll do everything I can to get in direct contact with the Web Store team. I can't really do that on behalf of the original publisher (boukestam).

Also, just curious, @JayTeeHaitch ... which ads are you seeing? Or more specifically, where are you seeing them? I only get them when I have the category tabs enabled, which I personally prefer to do without (see the `Recommended Gmail Settings` section of the README), since the bundling feature can handle categories. There's also the option to use an ad blocker like uBlock Origin. I'm always surprised to meet someone who doesn't already have that installed. 😅

**Comment by RedcoatAsher** [comment]
@JayTeeHaitch -- cc: @richardfrost @russelldc 
I posted about this before, somewhere else, but do not remember the link. So, here is how anyway...

- install [Stylus Chrome Extension](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne)
- create a rule in Stylus for `mail.google.com` w/ the [contents of this Gist](https://gist.github.com/RedcoatAsher/dfd34688d4f139095d86c5ab34d852f3)
...and BAM! no ads!!

Also, if interested, check out my [Dark Mode for Google News Gist](https://gist.github.com/RedcoatAsher/d849aa4e103a90ebd156bbf69273497b). Steps are the same as above except for `news.google.com`
